code/__HELPERS/game.dm

```diff
@@ -485,6 +485,68 @@
 			++i
 	return L
 
+/proc/pollCultists(var/mob/living/Nominee) // Cult Master Poll
+	var/time_passed = world.time
+	var/list/yes_voters = new
+	var/list/cult_total = new
+	if(world.time<1)
+		Nominee << "It would be premature to select a leader while everyone is still settling in, try again in [round((1800-world.time)/10)] seconds"
+		return
+	for(var/datum/mind/B in SSticker.mode.cult)
+		if(!isliving(B.current))
+			continue
+		var/mob/living/M = B.current
+		if(isliving(M))
+			M << 'sound/hallucinations/im_here1.ogg'
+			M.verbs -= /mob/living/proc/cult_master
+			to_chat(M, "<span class='cultlarge'>Acolyte [Nominee] has asserted that they are worthy of leading the cult. A vote will be called shortly.</span>")
+	sleep(250)
+	for(var/datum/mind/B in SSticker.mode.cult)
+		if(!isliving(B.current))
+			continue
+		var/mob/living/M = B.current
+		M << 'sound/magic/exit_blood.ogg'
+		switch(askuser(M,"[Nominee] seeks to lead your cult, do you support  them?","Please answer in 20 seconds!","Yes","No","Abstain", StealFocus=0, Timeout=200))
+			if(1)
+				if((world.time-time_passed)>500)
+					M << "<span class='danger'>Sorry, your vote came too late!</span>"
+					M << 'sound/machines/buzz-sigh.ogg'
+				else
+					M << "<span class='notice'>Choice registered: Yes.</span>"
+					yes_voters += M
+					cult_total += M
+			if(2)
+				if((world.time-time_passed)>500)
+					M << "<span class='danger'>Sorry, your vote came too late!</span>"
+					M << 'sound/machines/buzz-sigh.ogg'
+				else
+					M << "<span class='danger'>Choice registered: No.</span>"
+					cult_total += M
+			if(3)
+				M << "<span class='danger'>Choice registered: Abstain.</span>"
+	sleep(300)
+	if(yes_voters.len > (cult_total.len/2))
+		var/datum/action/innate/cultmast/finalreck/FinalReckoning = new()
+		var/datum/action/innate/cultmast/cultmark/Mark = new()
+		FinalReckoning.Grant(Nominee)
+		Mark.Grant(Nominee)
+		Nominee.mind.special_role = "Cult Master"

```

Comment:
I figured it was extremely unlikely and not a big issue if it happened. If the cult just elected a guy who can't stay alive 3 minutes into a round then you've got bigger things to worry about. 

But if this would cause runtimes or something  ill add some checks.